plumb in log shipping to logs.gutools.co.uk

steppenwells opened the pull request:
code clearly cribbed from tagmanager.

still need to deploy to code and verify in the logs app.

steppenwells committed:
plumb in log shipping to logs.gutools.co.uk

clloyd commented on cloudformation/campaign-central.json:
```diff
@@ -185,6 +185,23 @@
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Action": [ "kinesis:*" ],
```
I know we always do `kinesis:*` but is this an opportunity to pass a specific kinesis stream?

clloyd commented:
👍

steppenwells committed:
only allow kinesis access to logs kinesis stream

steppenwells committed:
Merge remote-tracking branch 'origin/master' into logshipping

Conflicts:
	build.sbt

steppenwells committed:
add log message to ga data fetch.

not very important but gives us something to verify logging is working correctly

steppenwells committed:
correct join used to generate log shipping arn

adds debug bootstrap logging

steppenwells committed:
Merge remote-tracking branch 'origin/master' into logshipping

steppenwells committed:
add access to configuration bucket

steppenwells committed:
give logging more time to bootstrap

steppenwells committed:
use explicit log configuration

rather than the default.

removes aka logging bootstrap - it didn't help
